projects/igniteui-angular/src/lib/icon/icon.service.ts

```diff
@@ -268,15 +279,15 @@ export class IgxIconService {
     public addSvgIconFromText(
         name: string,
         iconText: string,
-        family = "",
+        family = this._defaultFamily.name,
```

Comment:
Was going to ask about this, but saw it was merely the same as the inner `cacheSvgIcon` would do and matches `addSvgIcon`.
Bit awkward to funnel unassigned into the material collection, might be better if that was just a requirement but prob too late for that to change now.